packages/ffe-form-react/src/HorizontalSelector.js

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { array, object, string, func } from 'prop-types';
+
+const HorizontalSelector = ({
+    options,
+    initialSelected,
+    label,
+    description,
+    onChange,
+    field,
+    ...rest
+}) => {
+    const [selectedValue, setSelectedValue] = useState(
```

Comment:
Du trenger vell ikke denne staten alls. Du vet ju vilken input som er checked

`input[type="radio"]:checked+label{ background; red } `

Dette trenger ju gjør om markupen litt

husk for på labeln når du flytter ut input
```
<input type="radio" id="bladibladi" />
<label for="bladibladi">lorum</label>
```

Reply:
Er det noen grunn til å foretrekke css for å holde styr på optionen som er valgt ?

Ser at det kan fjernes. Men det må bare legges til med engang, man ønsker å gjøre noe bortsett fra å style den valgte.

Reply:
Vad mener du? Ikke sikkert på att jag skjønner spørsmålet men med min løsning her så undgår du din interna state og en extra class før om ett element er valgt. Vafor ha en egen state og en class for noe som man kan få gratis med html/css?